Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -266,6 +266,7 @@ extension XCTestCase {
 }
 
 extension ConfigurationTests {
+    // swiftlint:disable:next void_return
     static var allTests: [(String, (ConfigurationTests) -> () throws -> Void)] {
```

Comment:
I think types in the middle could be either `()` or `Void`, and to adapt both `void_return` and `empty_parameters` would just ignore types in the middle.